File: exercicios/R/1961/1961.r
input = file('stdin','r')

flag = as.integer(0)
sum = as.integer(0)

line = strsplit(readLines(input,n=1), " ")

x = as.integer(line[[1]][1])
y = as.integer(line[[1]][2])

a <-c(y)

line2 = strsplit(readLines(input,n=1), " ")

for(i in 1:y){
    a[i] = as.integer(line2[[1]][i])
}

for(j in 2:y){
    if(a[j] > a[j-1]){
        sum = (a[j] - a[j-1])
    }else{
        sum = (a[j-1] - a[j]);
    }

    if(sum <= x){
        flag = 1
    }else{
        flag = 0
        break
    }
}
if(flag == 1){
    write("YOU WIN",'');
}else{
    write("GAME OVER",'');}

File: exercicios/JavaScript/1961/1961.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var val1 = lines.shift().split(' ')

var x = parseInt(val1[0])
var y = parseInt(val1[1])

var val = lines.shift().split(' ')

var a = []
var sum = 0;
for(let i=0; i<y; i++){
    a[i] = val[i]
}

for(let i=1;i<y;i++)
{
    if(a[i]>a[i-1])
    sum = (a[i] - a[i-1]);
    else
    sum = (a[i-1] - a[i]);
    if(sum<=x)
    flag=1;
    else
    {
        flag=0;
        break;
    }
}
if(flag==1){
    console.log("YOU WIN");
}else{
    console.log("GAME OVER");
    }
